app/customer.py

```diff
@@ -0,0 +1,76 @@
+import math
+from dataclasses import dataclass
+
+
+from app.shop import Shop, print_check
+
+
+@dataclass
+class Customer:
+    name: str
+    product_cart: dict
+    location: list
+    money: int
+    car: dict
+
+
+def cost_of_km(customer: Customer, data: dict) -> int | float:

```

Comment:
This method logically belongs to class Car. Also you can pass here data["FUEL_PRICE"] - as a single element, no need to pass entire dict